subiquity/cmd/server.py

```diff
@@ -26,6 +27,24 @@
     )
 
 
+class CommandLineParams:
```

Comment:
I'd appreciate to have a `__str__` or `__repr__` defined so we can log the content of a CommandLineParams object without going through `.__dict__`
Would leaning on `attr.s` or `dataclass` make sense?

Reply:
Thanks!